dev-support/create-release.sh

```diff
@@ -141,4 +141,7 @@ echo "The artifacts for this run are available at ${ARTIFACTS_DIR}:"
 run ls -1 ${ARTIFACTS_DIR}
 echo 
 echo "Remember to sign them before staging them on the open"
-echo
+echo     
```

Comment:
whitespace:: /Users/aw/Src/drd-github/patchprocess/whitespace-eol.txt end of line